libs/shared/ui/src/lib/components/help-section/help-section.tsx

```diff
@@ -14,16 +14,23 @@ export function HelpSection(props: HelpSectionProps) {
     <div data-testid="help-section" className={`py-8 px-10 border-t border-neutral-200 ${className}`}>
       <p className="text-neutral-400 text-sm mb-5">{description}</p>
       {links &&
-        links.map((link, i, row) => (
-          <Link
-            key={i}
-            className={`font-medium ${i + 1 === row.length ? '' : 'mb-2'}`}
-            link={link.link}
-            linkLabel={link.linkLabel || link.link}
-            external={link.external}
-            iconRight={IconAwesomeEnum.ARROW_UP_RIGHT_FROM_SQUARE}
-          />
-        ))}
+        links.map((link, i, row) =>
+          link.external ? (
+            <ExternalLink key={i} color="sky" className={`flex ${i + 1 === row.length ? '' : 'mb-2'}`}>
```

Comment:
use flex gap for margin

Reply:
it comes from the old code, I can update it if needed

Reply:
https://github.com/Qovery/console/pull/864/commits/8b0877172944f686d46a03ec1ea99dd46060432a https://github.com/Qovery/console/pull/864/commits/86e325494555c1b281ec52d5f98122e4e7a55951